x/deployment/types/id.go

```diff
@@ -50,5 +59,11 @@ func (id GroupID) Equals(other GroupID) bool {
 
 // Validate method for GroupID and returns nil
 func (id GroupID) Validate() error {
+	if id.Owner.Empty() {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "owner cannot be empty")
+	}
+	if id.DSeq == 0 && id.GSeq == 0 {
```

Comment:
`return id.DeploymentID().Validate()`

Reply:
ditto, I can't because `GSeq` is not defined for `DeploymentID`

Reply:
`GroupID = DeploymentID + GSeq`; `Validate(GroupID) = Validate(DeploymentID) + Validate(GSeq)`